app/javascript/actions/gameActions.js

```diff
@@ -1,8 +1,8 @@
 import axios from 'axios';
 import 'dotenv/config';
 
-    const  axiosInstance = ({path, method = 'post', data}) => {      
-      return axios( {url:process.env.BASE_URL+path, method, data} )
+    const  axiosInstance = ({path, method = 'post', data}) => {  
+      return axios( {url:path, method, data} )
```

Comment:
Now that you are only passing in the path here, where is the base URL referenced?